lib/active_merchant/billing/gateways/shift4_v2.rb

```diff
@@ -57,6 +57,37 @@ def add_amount(post, money, options, include_currency = false)
         super
         post[:currency]&.upcase!
       end
+
+      def add_creditcard(post, payment_method, options)
```

Comment:
If this method is for adding a bank account, can we change the method name from `add_creditcard` to `add_bank_account`?

Reply:
I had the same consideration, but I preferred to keep the method signature in the same way that SecurionPay does, so I can properly use the `super` keyword and be more explicit about the implementation